pxr/usdImaging/usdImaging/delegate.h

```diff
@@ -59,6 +59,7 @@
 #include "pxr/base/tf/declarePtrs.h"
 #include "pxr/base/tf/hashmap.h"
 #include "pxr/base/tf/hashset.h"
+#include "pxr/base/tf/denseHashset.h"
```

Comment:
Hey @marktucker, I think that needs to be denseHashSet.h -- capital 'S'

Reply:
It sure does. Unless you're on Windows. Sigh. Fixed.